terraform/gcp/modules/cloud_run_v2/variables.tf

```diff
@@ -0,0 +1,52 @@
+variable "service_name" {
+  type    = string
+  default = "default-service"
+}
+
+variable "make_public" {
+    type = bool
+    default = false
+  
```

Comment:
can you please remove all the empty lines? Same front the other files too please.